spec/upload_to_browserstack_app_live_action_spec.rb

```diff
@@ -1,5 +1,11 @@
 describe Fastlane::Actions::UploadToBrowserstackAppLiveAction do
   describe 'Upload to BrowserStack AppLive' do
+
```

Comment:
@iBotPeaches Can you remove this line? 
It's violating the rubocop rules 

```
Inspecting 12 files
....C.......

Offenses:

spec/upload_to_browserstack_app_live_action_spec.rb:3:1: C: Extra empty line detected at block body beginning.

12 files inspected, 1 offense detected
```

Reply:
Passes locally now.

```
Finished in 1.28 seconds (files took 1.62 seconds to load)
23 examples, 0 failures

Coverage report generated for RSpec to /Volumes/Nexus/Github/browserstack-fastlane-plugin/coverage. 210 / 227 LOC (92.51%) covered.
Running RuboCop...
Inspecting 12 files
............

12 files inspected, no offenses detected
➜  browserstack-fastlane-plugin git:(support-fallback-aab) ✗ 
```